client-app/ui-kit/components/atoms/container/vc-container.vue

```diff
@@ -8,10 +8,14 @@
 
 <style lang="scss">
 .vc-container {
-  @apply grow pt-6 pb-8 bg-[color:var(--color-body-bg)];
+  @apply grow bg-neutral-50 p-6 md:pb-9 lg:px-8 2xl:px-12;
 
   &__wrapper {
-    @apply mx-auto max-w-screen-2xl lg:px-5 2xl:px-18;
+    @apply mx-auto max-w-[87.75rem];
+
+    & .vc-widget--size--lg {
```

Comment:
Component shouldn't change other component styles event if they are not scoped

Reply:
No, one component can affect another component because they are part of the same system. But I can write differently.

Reply:
As I said before, there is weird layout here. So I'd prefer to wait for a qa opinion.